simapp/utils.go

```diff
@@ -81,7 +81,7 @@ func CheckExportSimulation(
 			return err
 		}
 
-		if err := ioutil.WriteFile(config.ExportStatePath, []byte(appState), 0644); err != nil {
+		if err := ioutil.WriteFile(config.ExportStatePath, []byte(appState), 0600); err != nil {
```

Comment:
Where does this change come from? Why aren't creating this file with 0644 perms?

Reply:
Linting via 2bdf255

Reply:
Yeah I see, the good @marbar3778 did the right thing in shutting down the linter here :)
Yet I think we could just revert this and just put a `//nolint: gosec` statement here and that would sort it out and at the same time give users a slightly better UX